shared/src/business/useCases/courtIssuedDocument/fileAndServeCourtIssuedDocumentInteractor.ts

```diff
@@ -71,25 +71,37 @@ export const fileAndServeCourtIssuedDocumentInteractor = async (
 
   const subjectCaseEntity = new Case(subjectCase, { applicationContext });
 
-  const docketEntryToServe = subjectCaseEntity.getDocketEntryById({
+  const originalSubjectDocketEntry = subjectCaseEntity.getDocketEntryById({
```

Comment:
📓 Just a note that this was renamed from `originalSubjectDocketEntry` to `docketEntryToServe` as part of the #9681 refactor, this branch may encounter a merge conflict with staging if that story is merged first.

Reply:
yeah, I caught that with the unit test. I'll let y'all merge first and fix this after.